redirects/14.0.txt

```diff
@@ -40,6 +40,10 @@ applications/finance/accounting/payables/supplier_bills/manage.rst applications/
 applications/general/calendars/outlook/outlook_calendar.rst applications/productivity/calendar/outlook.rst           # general/calendars/outlook/outlook_calendar -> productivity/calendar/outlook
 applications/general/calendars/google/google_calendar_credentials.rst applications/productivity/calendar/google.rst  # general/calendars/google/google_calendar_credentials -> productivity/calendar/google
 
+# applications/hr
+
+applications/hr/payroll/configuration.rst applications/hr/payroll.rst  # applications/hr/payroll/configuration.rs --> applications/hr/payroll.rst
```

Comment:
not necessary for redirect functionality, just caught a small typo
```suggestion
applications/hr/payroll/configuration.rst applications/hr/payroll.rst  # applications/hr/payroll/configuration.rst --> applications/hr/payroll.rst
```

Reply:
TY! Fixed and trying to merge this now.